packages/botonic-react/src/util/logs.js

```diff
@@ -0,0 +1,17 @@
+import { getProperty } from '../utils'
+
+export function warnDeprecated(legacyCode, newCode) {
+  console.warn(`"${legacyCode}" is deprecated. Use "${newCode}" instead.`)
+}
+
+export function warnDeprecatedProps(props) {
+  const legacyNewProps = [
+    { legacy: 'enableTimestamps', new: 'enabletimestamps' },
+    { legacy: 'imageStyle', new: 'imagestyle' },
+  ]
+  legacyNewProps.forEach(
+    p =>
+      getProperty(props, p.legacy) !== undefined &&
```

Comment:
I think that st like this is more clear
```
if (getProperty(props, p.legacy)) {
   warnDeprecated(p.legacy, p.new)
}
```

Reply:
changed, thanks! 👍 , we have to be sure that is `undefined` because if somebody is using `enableTimestamps: 0` it won't prompt the error